lib/sedna/backend/mindspore/__init__.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 The KubeEdge Authors.
```

Comment:
thanks for your great job! BTW, should we provide some example about mindspore to end user? joint inference or incremental learning?

Reply:
OK, I'll upload a resnet example by tomorrow

Reply:
@chou-shun please put your example in `lib/examples/backend/mindspore/`.